origins_workflow/tests/src/Nightwatch/Pages/ModerationSidebar.js

```diff
@@ -40,7 +40,8 @@ module.exports = {
     // See fieldSubThemeWithMotoringValue vs moderation state dropdown below as examples.
     createArticleNode: function(workflowState) {
       return this
-        .setValue('@title', this.props.titleText + ' (' + workflowState + ')')
+        .setValue('@title', [Date.now() + ' -- ' + this.props.titleText + ' (' + workflowState + ')', this.api.Keys.TAB])
```

Comment:
Ensures you can re-run this test without falling foul of the unique title validation constraint.